backend/src/models/Post.js

```diff
@@ -4,6 +4,12 @@ export default {
   id: { type: 'string', primary: true, default: uuid },
   activityId: { type: 'string', allow: [null] },
   objectId: { type: 'string', allow: [null] },
+  image: {
+    type: 'relationship',
+    relationship: 'TEASER_IMAGE',
```

Comment:
It is called `TeaserImage` because that is what it was called in the old `alpha` system, and I copied the naming convention without thinking about whether it should be renamed. 
It does however kinda make sense to me cause the image teases what the article/post is about, at least that is the way some use the image. 
Having said that, I have no issue renaming it, and `HeroImage` would be a good choice, in my mind.